terraform/modules/database/database.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_rds_cluster_instance" "cluster_instances" {
+  count                        = "${var.db_instance_count}"
+  identifier                   = "query-cluster-${var.deployment_stage}-${count.index}"
+  cluster_identifier           = "${aws_rds_cluster.query.id}"
+  instance_class               = "db.r4.large"
+  publicly_accessible          = "true"
```

Comment:
If possible, this should be private. We have PgBouncer in front of this, which test clients can connect to. Eventually, none of the infra should be publicly available.

Reply:
👍